### DISCUSSION TITLE
Virustotal flags

### DISCUSSION THREAD

**Discussion by fdfytr** [ohmyzsh/ohmyzsh#11442] [Help] [answered]
Just curios why 2 vendors show alerts for install.sh file

**Comment by rwmitchell** [comment]
Because they’re crap?

what vendors?  What did they flag?  Did they give any other info?

without details, this is useless.

**Reply by fdfytr** [reply]
[Now it is just one](https://www.virustotal.com/gui/url/ea1a567e881e4d967c80b0f0bd1ee0fec6cf653c0f9594279475a8aa05de9767/detection) Before my post CMC Threat Intelligence was 2nd

**Reply by mcornella** [reply]
@rwmitchell there is no need to act defensive and use words such as useless (which it wasn't btw, there was plenty to check). Don't make me tap the code of conduct sign.

**Reply by rwmitchell** [reply]
The initial post didn't provide a link to further information on the issue, it only insinuated OMZ couldn't be trusted. 

If the install script had been changed, those changes would have previously been posted to discord or the github repo would had to have been compromised in a way that circumvented GitHub itself.

**Reply by mcornella** [reply]
You're reading malicious intent on a short phrase that says "just curious". Not at all necessary and it isn't the first time you're being hostile.

And it was plenty information for me, it was either scanning the URL or the file, and I saw easily what they were referring to.

**Comment by mcornella** [comment, marked as answer]
The current analysis for the install.sh URL shows only 1 currently, as I rerun the analysis. The previous results were from an analysis from 2022.

In any case, these results are false positives, as there is no possible malware coming from a url that serves a plain text file.

The proper analysis should be done on the actual script file, which you can find [here](https://www.virustotal.com/gui/file/73bc685bfac98e4d41c41d49b2d4d7bbcf55cbac7a3dbb4022073410ff15fd03/summary). You'll see that 0 positives are returned.

You can also inspect the install.sh, which is very straightforward, it only runs git and sets up zsh.

Let me know if you have any other doubts, and thanks for reporting this.
